pages/learn/integration/pre-built-integrations/kafka-to-salesforce-integration.js

```diff
@@ -227,14 +227,14 @@ export default function Learn({ samples, content }) {
                     <p>In enterprises, business data like inventory levels, order statuses, or product prices change
                       constantly, and multiple entities require access to these updates. Utilizing messaging systems
                       like Kafka enables the real-time consumption of this information by multiple parties. Keeping
-                      Salesforce updated with frequently changing information, such as current pricing, is vital for
+                      Salesforce updated with frequently changing information such as current pricing is vital for the
```

Comment:
IMO, using commas makes it more readable. Prefer to keep those, if this is not wrong.

Reply:
It is grammatically not correct to have the comma in this case as it is added for a restrictive clause. The explanation copied from [1] is as follows.

When to Use Such As Without a Comma
Don’t use a comma if such as is part of a restrictive clause.

Trees such as oaks and elms don’t grow at this altitude.
Here we have a restrictive phrase. Try taking it out of the sentence: Trees don’t grow at this altitude. The sentence isn’t true anymore. We’re not trying to say that no trees grow at this altitude—we’re restricting the statement to specific types of trees: oaks and elms.

[1] https://www.grammarly.com/blog/such-as-comma/